wiggum/labeled_dataframe.py

```diff
@@ -168,11 +169,29 @@ def __init__(self,data=None,meta=None,results=None):
             results = os.path.join(data,result_csv)
             trends = os.path.join(data,trend_json)
             data = os.path.join(data,data_csv)
+        elif type(data) == list and meta ==None and results ==None:
+            # if files are uploaded from wiggum-app
+            filename = ''
+            for file in data:
+                filename = file.filename
+                if ('/' in filename):
+                    # if file name contains '/', Wiggum runs in Windows system
+                    # remove the directory and only use the file name
+                    filename = filename.split('/')[1]
+
+                if filename == data_csv:
+                    data = file
+                elif filename == meta_csv:
+                    meta = file
+                elif filename == result_csv:
+                    results = file
+                elif filename == trend_json:
+                    trends = file
```

Comment:
is there no way to do this on the app side?  like have the upload step put the files in a temp directory and then pass that to the initialize function?

Reply:
That will be more complicated.

Reply:
but I think better and possibly more on the path to allowing the user to download and save their results where they would like.